### ISSUE TITLE
Do you feel supporting "reusable holdout" is in scope of prettytensor?

### ISSUE THREAD

**Issue by windenntw** [google/prettytensor#11]
Given that prettytensor combined with TF and ipython makes very very easy to do large scale, iterative exploration, I wonder if it would be in scope to incorporate a default implementation of Reusable Holdout [1] into the runner class and provided examples -- specially for less-experienced people to get a head-start in doing analysis in a good way.

[1] http://googleresearch.blogspot.com/2015/08/the-reusable-holdout-preserving.html

**Comment by eiderman** [comment]
Yes, I think that an implementation would be good to have. The method mentioned in the paper (thresholdout) requires a full pass over both the training and holdout dataset so it would need to be implemented as part of the runner or queuing system. Utilities for splitting the data and tracking the number of uses would also be nice.

Would you be interested in submitting a patch? You would need to follow the instructions at https://github.com/google/prettytensor/blob/master/CONTRIBUTING.md
